recipes/trino-server/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "trino-server" %}
+{% set version = "433" %}
+{% set java_version = 17 %}
```

Comment:
Please either write this without Jinja (preferred) or put the value in a `conda_build_config.yaml`. Avoid top-level Jinja in `meta.yml` recipes as much as possible.